tools/devctr/Dockerfile

```diff
@@ -60,12 +62,21 @@ RUN apt-get update \
         requests \
         requests-unixsocket \
         retry \
+    # install cross-compiled fdt library
```

Comment:
Please add a issue to remove the libfdt once we have a working CI for ARM.

Reply:
Opened #1078.